Tasks/distinct.js

```diff
@@ -1,16 +1,14 @@
-// Deduplicate values in array
+'use strict';
 
-DISTINCT =data=> {
-    A=new Set(); w=0;
-    data.forEach((a) => {
-        if (A.has(a)) {
-        data.splice(w, 1)
-        } else {
-        A.add(a)
-        };
-        w++;
-    });
-  return data;}
+const distinct = (array) => {
+  const result = [...array];
```

Comment:
> * Prefer for..of
> * Don't change incoming parameters

I changed result, is it parameter? this way create new array and copy each element, I tried array === result and it showed false.